src/__tests__/mockups/error.mock.ts

```diff
@@ -5,6 +5,18 @@ export const error = {
     description: 'Bad request.',
     isOperational: true,
   },
+  401: {
+    name: 'Error',
+    httpStatusCode: 401,
+    description: 'Must be authenticated.',
+    isOperational: true,
+  },
+  403: {
+    name: 'Error',
+    httpStatusCode: 403,
+    description: "You don't have the right access to execute this request.",
```

Comment:
Wording : "the access right" instead of "the right access"

Reply:
You don't have the right access to execute this request. : Tu n'as pas le bon accès pour exécuter cette requête.
You don't have the access right to execute this request.:  Tu n'as pas le bon droit d'accès pour exécuter cette requête.

Rien de mauvais dans les deux phrases.

Reply:
Ok sorry about it